client/repo.go

```diff
@@ -16,14 +17,22 @@ import (
 // (This is normally defaults to "~/.notary" or "~/.docker/trust" when enabling
 // docker content trust).
 func NewNotaryRepository(baseDir, gun, baseURL string, rt http.RoundTripper,
-	retriever notary.PassRetriever, trustPinning trustpinning.TrustPinConfig) (
+	retriever notary.PassRetriever, trustPinning trustpinning.TrustPinConfig, UseNative bool) (
```

Comment:
makes sense, changed it. Should getUseNative change as well or should we keep that as is to keep with the config of the functions in tuf.go?